_data/simple-icons.json

```
@@ -372,7 +372,7 @@
         },
         {
             "title": "Air India",
-            "hex": "D9000D",
+            "hex": "DD2842",
             "source": "https://www.airindia.com"
```

If sourced from Wikipedia, I'd update this entry to that for now.
```suggestion
            "source": "https://commons.wikimedia.org/wiki/File:Air_India_2023.svg"
```

Looks like other than the specific 'rebrand' page - they've not yet updated their main website.

---

Shall we keep this PR and any changes on hold till December 2023?

---

Can do - is that when the rebrand is supposed to take effect?